src/components/Details/Legislation/LegislationIntroduction.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from "react";
+import MatterStatus from "../../../models/MatterStatus";
+import IndexedMatterGram from "../../../models/IndexedMatterGram";
+import { DecisionResult } from "../../Shared";
+
+export interface LegislationIntroductionProps {
+  matterStatus: MatterStatus;
+  indexedMatterGrams: IndexedMatterGram[];
+}
+
+const LegislationIntroduction: FC<LegislationIntroductionProps> = ({
+  matterStatus,
+  indexedMatterGrams,
+}: LegislationIntroductionProps) => {
+  return (
+    <div>
+      <h1 className="mzp-u-title-sm">{matterStatus.matter?.name}</h1>
+      <b className="mzp-u-body-sm">{matterStatus.matter?.title}</b>
+      <div
```

Comment:
I don't think `mzp-u-body-sm` is valid Mozilla protocol CSS class

Reply:
where can I find a list of valid classes?  I use their website but in a case like this where I want something that's just a little bit different from the listed items, it becomes very difficult to figure out what is valid and what is not.

Reply:
Not sure if there's a page that lists them. I didn't find the class inside `node_modules/@mozilla-protocol/core/css`

Reply:
https://protocol.mozilla.org/docs/naming.html

Here's how they name their class though